hosting_docs/source/installation/go-live-checklist.rst

```diff
@@ -0,0 +1,18 @@
+.. _go-live-checklist:
+
+Go Live Checklist
+=================
+
+There are many things to be setup correctly before your CommCareHQ instance is made accessible over the internet. This checklist helps you to plan a new environment setup and make sure that you have everything ready to go live.
```

Comment:
I don't have a good feeling about where to set my inner grammar nazi :upside_down_face: so please tell me if I ought to dial it down (or if you'd like me to dial it up :grimacing: :speak_no_evil: ).

You've got "setup" twice in this paragraph; the first time you've used it as a verb ("to be setup"), and the second time as a noun ("a new environment setup"). The second time is correct, "setup" is a noun. But the first time is incorrect, because the verb is "set", and "up" is a preposition. This becomes clear when you think about the "-ing" form ([present participle](https://dictionary.cambridge.org/dictionary/english/present-participle)) of the verb: It's "setting up", not "setupping".

```suggestion
There are many things to be set up correctly before your CommCareHQ instance is made accessible over the internet. This checklist helps you to plan a new environment setup and make sure that you have everything ready to go live.
```

There are five more "setups" in this commit, and they're all verbs. I would feel like a bit of a jerk if I pointed out each one, so I won't do that.

(There is a "BUT" here. English is not like Python, which has a [steering council](https://www.python.org/dev/peps/pep-8016/) that decides what is and what is not Python. English is whatever consensus among English speakers says it is. If enough people decide that "setup", "setupping" and "setupped" are correct, then they are correct.)

Reply:
Thank you very much, I have updated this!